### PR TITLE
Update develop scripts due to funds and assets rake task

### PR DESCRIPTION
**Pull request by paoolo**

### CODE REVIEW

File: spec/dummy/bin/prepare-environment [hunk 1]
```diff
@@ -9,7 +9,9 @@ pushd $(dirname $(dirname $(dirname ${__dir})))
 popd
 
 pushd $(dirname ${__dir})
+    bundle exec rake assets:precompile
```
Comment (paoolo) [inline spec/dummy/bin/prepare-environment]: @mkasztelnik I guees that if there is no ``config.assets.compile = false`` in ``spec/dummy/config/environments/development.rb`` like in ``spec/dummy/config/environments/production.rb`` then after making some commands using ``bundle exec rake`` will automagic recompile assets by default?

Comment (mkasztelnik) [inline spec/dummy/bin/prepare-environment]: Yeap

**Comment by paoolo** [comment]
Waiting for +1

File: .gitignore [hunk 2]
```diff
@@ -31,4 +31,7 @@
 /spec/dummy/tmp
 /.project
 nbproject/
-coverage
\ No newline at end of file
+coverage
+
+# Ignore assets
+spec/dummy/public/assets/
```
Comment (mkasztelnik) [inline .gitignore]: This also can be removed since assets will not be generated using rake task.

**Comment by mkasztelnik** [comment]
After solving issue reported above :+1: from me :smile:

**Comment by paoolo** [comment]
Thanks. Waiting for second +1

**Comment by nowakowski** [comment]
:+1:

**Pull request merged**